src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/PriceRangeFilter.tsx

```diff
@@ -21,7 +19,7 @@ export const PriceRangeFilter: React.FC<{
         const minStr = min === FilterState.MIN_PRICE ? "*" : min
         const maxStr = max === FilterState.MAX_PRICE ? "*" : max
 
-        filters.setFilter("price_range", `${minStr}-${maxStr}`, mediator)
```

Comment:
Nice cleanup getting rid of mediator throughout.

Reply:
I plan to go back and do the same for the aforementioned PR's.

Reply:
Unrelated (but also very much related), it would be so nice to consolidate all of our `ArtworkFilters` into one; Matt and I have talked about this a bunch as we now have three duplicated instances in the Reaction.

Reply:
That sounds like a fun thing to pair on some time. 😃🍐